specification/webpubsub/data-plane/WebPubSub/stable/2023-07-01/webpubsub.json

```diff
@@ -0,0 +1,1821 @@
+            },
+            "collectionFormat": "multi"
+          },
+          {
+            "in": "query",
+            "name": "minutesToExpire",
+            "description": "The expire time of the generated token.",
+            "type": "integer",
+            "format": "int32",
+            "default": 60,
+            "minimum": 1
+          },
+          {
+            "in": "query",
+            "name": "api-version",
+            "description": "The version of the REST APIs.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "in": "query",
+            "name": "group",
+            "description": "Groups that the connection will join when it connects.",
+            "type": "array",
+            "items": {
+              "type": "string"
+            },
+            "collectionFormat": "multi"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/ClientTokenResponse"
+            }
+          },
+          "default": {
+            "description": "Error response",
+            "schema": {
+              "$ref": "#/definitions/ErrorDetail"
+            },
+            "x-ms-error-response": true,
+            "headers": {
+              "x-ms-error-code": {
+                "x-ms-client-name": "ErrorCode",
+                "type": "string"
+              }
+            }
+          }
+        },
+        "x-ms-examples": {
+          "WebPubSub_GenerateClientToken": {
+            "$ref": "./examples/WebPubSub_GenerateClientToken.json"
+          }
+        }
+      }
+    },
+    "/api/hubs/{hub}/:removeFromGroups": {
+      "post": {
+        "tags": [
+          "webpubsub"
+        ],
+        "summary": "Remove filtered connections from multiple groups.",
+        "operationId": "WebPubSub_RemoveConnectionsFromGroups",
+        "consumes": [
+          "application/json-patch+json",
+          "application/json",
+          "text/json",
+          "application/*+json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "hub",
+            "description": "Target hub name, which should start with alphabetic characters and only contain alpha-numeric characters or underscore.",
+            "required": true,
+            "type": "string",
+            "pattern": "^[A-Za-z][A-Za-z0-9_`,.[\\]]{0,127}$"
+          },
+          {
+            "in": "query",
+            "name": "api-version",
+            "description": "The version of the REST APIs.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "in": "body",
+            "name": "message",
+            "description": "Target groups and connection filter.",
+            "schema": {
+              "$ref": "#/definitions/RemoveFromGroupsRequest"
+            }
```

Comment:
Should this body param be `required: true`?

Reply:
Yes. Marked it to `required:true`